src/helpers/visualizationType.ts

```diff
@@ -29,3 +29,11 @@ export async function getVisualizationTypeFromVisualizationClass(
 
     return type;
 }
+
+export function getVisualizationType(type: ChartType): ChartType {
```

Comment:
I suggest this name `transformComboChartType` or `transformComboType`.

Reply:
I think this name will make confusion because `type` prop can be any chart types as same as return type, not only for combo chart.